.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: r
+cache: packages
```

Comment:
travis doc says it should be this?
```
cache:
  packages: true
```

Reply:
Don't think so, see https://docs.travis-ci.com/user/caching/#r-package-cache

Reply:
ahh ok. looks good then.  i should have linked the example i saw, it was here: https://docs.travis-ci.com/user/languages/r/#packrat